apps/server/src/modules/learnroom/controller/rooms.controller.ts

```diff
@@ -63,7 +62,7 @@ export class RoomsController {
 	}
 
 	@Post(':roomId/copy')
-	@RequestTimeout(serverConfig().INCOMING_REQUEST_TIMEOUT_COPY_API)
+	@RequestTimeout('INCOMING_REQUEST_TIMEOUT_COPY_API')
```

Comment:
You removed `INCOMING_REQUEST_TIMEOUT_COPY_API` from the `InterceptorConfig` and `FileStorageConfig`. Where is this value coming from?

Reply:
Die Logik ist jetzt folgendermaßen:
Es kann theoretisch jeder envirement Name rein gereicht werden. Aber die Orte die diesen verwenden müssen in ihre config definieren das es den Wert auch gibt. Das Modul welches von dem app deployments genutzt wird (folder apps/*) muss die Envirements Variablen der Nest Config zuweisen. 

Mir fällt aber gerade auf das wir keine Guar haben ob das eine number ist.